nss-day-cohort-29/reactive-nutshell-springwynne
# Rr events

# Description

Implements functionality to display events.


## Type of change

- [x ] New feature (non-breaking change which adds functionality)


# Testing Instructions for Change Made

Checkout rr-events
Run nom start
Navigate to "Events" tab
You should see a list of events ordered by date and time.  The current users events have a delete button, the events of the users friends do not.


# Checklist:
